plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt

```diff
@@ -176,10 +173,8 @@ internal constructor(
         /**
          * Add 3rd party reporters.
          */
-        fun customReporters(configuration: Closure<NamedDomainObjectContainer<CustomReporter>>) {
-            // This method is needed for Groovy interop
-            // See https://discuss.gradle.org/t/multi-level-dsl-for-plugin-extension/19029/16
-            ConfigureUtil.configure(configuration, customReporters)
+        fun customReporters(configuration: Action<NamedDomainObjectContainer<CustomReporter>>) {
```

Comment:
Is this a binary API breaking change? IE. if projects are depending upon this plugin as a direct dependency rather than as a script, anything compiled against our APIs won't work anymore, correct?

Reply:
possibly... I know gradle has some stuff to auto convert between SAMs and Closures, but I'm not sure if it would help here

Reply:
in any case, this is the way is is supposed to be done in Gradle now, so I think at most, I'll make a note about it in release notes